app/src/main/java/ch/epfl/sdp/game/Game.java

```diff
@@ -132,12 +132,8 @@ public void addToDisplayList(Displayable displayable) {
      * @param displayable The displayable to be removed.
      */
     public void removeFromDisplayList(Displayable displayable) {
-        renderer.unDisplay(displayable);
-        if (displayable.isOnce()) {
-            displayablesOnce.remove(displayable);
-        } else {
-            displayables.remove(displayable);
-        }
+        displayable.unDisplayOn(mapApi);
+        displayables.remove(displayable);
```

Comment:
Here why not remove from displayablesOnce anymore?

Reply:
I thought displayablesOnce was the same as displayables

Reply:
No, they are different, displayablesOnce is something like items, which will only be displayed once, displayables are something like Enemies. Better check with Raphael about it, and if this will cause misunderstanding, maybe we need to refactor it.